MapboxNavigation/NavigationViewController.swift

```diff
@@ -422,7 +422,7 @@ open class NavigationViewController: UIViewController {
     
     // MARK: - NavigationViewController
 	
-    public func startNavigation(with route: Route, routeController: RouteController? = nil, locationManager: NavigationLocationManager? = nil) {
+    public func startNavigation(with route: Route, animated: Bool, routeController: RouteController? = nil, locationManager: NavigationLocationManager? = nil) {
```

Comment:
```suggestion
    public func startNavigation(with route: Route, animated: Bool = true, routeController: RouteController? = nil, locationManager: NavigationLocationManager? = nil) {
```

Reply:
I typically use default values only when people will almost always want that default value. 

To me this feels closer to 50/50, so I'd prefer to have people type it out rather than run the risk of people assuming it doesn't exist. For example, anyone using the navigation VC modally (e.g. any legacy users), will want this to be false.

Reply:
Okay makes sense, then let's keep it without the default parameter 👍